app/views/directives/pod-donut.html

```diff
@@ -4,16 +4,14 @@
   <span ng-if="!idled && total <= 99">
     {{total}}
   </span>
-  <span ng-if="idled">
-    Idle
-  </span>
 </div>
 <span ng-if="mini && total === 1 && !idled" class="donut-mini-text-name">pod</span>
 <span ng-if="mini && total !== 1 && !idled" class="donut-mini-text-name">
   <span ng-if="!idled && total > 99">
     {{total}}
   </span> pods
 </span>
+<span ng-if="idled" class="donut-mini-text-name">Idle</span>
```

Comment:
Given the siblings of this node also check for mini, I think it's probably best to do so here, too.

e.g., ng-if="mini && idled"

Reply:
Not only is it best, it's actually required.  :)  Note the stray "Idle" below "Idled due to inactivity."

![screen shot 2018-01-31 at 2 17 08 pm](https://user-images.githubusercontent.com/895728/35642584-82a2e7d0-0691-11e8-879d-c4876d95e3ac.PNG)